src/export/exporter.ts

```diff
@@ -86,4 +120,18 @@ export abstract class AzureMonitorBaseExporter implements BaseExporter {
 
     return filteredEnvelopes;
   }
+
+  private _sendFirstPersistedFile() {
```

Comment:
I wonder if we should separate persistence logic into it's own module. Currently, it is too tightly coupled with the exporter. This will make it difficult and messy to define storage related configurations later on.